src/renderer/App.js

```diff
@@ -10,19 +10,20 @@ import {
 import views from './views';
 import './css/App.css'
 import './css/main.css'
+import NewFeed from "./views/NewFeed";
 
 export class App extends React.PureComponent {
   render() {
     return (
       <div>
-        <Components.Topbar/>
```

Comment:
Could you make two route paths instead of replacing the watch page path? i.e `<Route path="/" exact>` The exact keyword means it will match if the path is exactly the root directory, vs any sub path from the directory. That way multiple pages can coexist